cap/modules/experiments/utils/das.py

```diff
@@ -31,8 +31,7 @@
         "doc": {
```

Comment:
Let's make the search case-insensitive, this will help:
```suggestion
     'mappings': {
        "doc": {
            "properties": {
                "name": {
                    "type": "keyword",
                    "normalizer": "lowercase_normalizer"
                }
            }
        }
    },
    "settings": {
        "analysis": {
            "normalizer": {
                "lowercase_normalizer": {
                    "type": "custom",
                    "char_filter": [],
                    "filter": "lowercase"
                }
            }
        }
    }

```